com/metacube/SCF/Assignment8/ZooOperations.java

```diff
@@ -0,0 +1,149 @@
+package GET2018.com.metacube.SCF.Assignment8;
+
+import java.util.Scanner;
+
+public class ZooOperations {
+
+	void addAnimal(AnimalType animalSpecies)
+    {
+        switch(animalSpecies)
+        {
+        case LION:
+            addLion();
+            break;
+        case CROCODILE:
+            addCrocodile();
+            break;
+        case PEACOCK:
+            addPeacock();
+            break;
+        default:
+        	break;
+        }
+        
+    }
+    
+    Scanner scan = new Scanner(System.in);
+    void addLion()
+    {
```

Comment:
Return type of this method should be Animal. The addAnimal method should return some animal. That animal will be then added to appropriate list.